selfdrive/ui/translations/languages.json

```diff
@@ -1,5 +1,6 @@
 {
   "English": "main_en",
+  "Nederlands": "main_nl",
```

Comment:
Do you mean the language or the country?
Because for language it should be 'Dutch'